shared/common-adapters/markdown/react.js

```diff
@@ -334,12 +335,19 @@ const reactComponentsForMarkdownType = {
     )
   },
   serviceDecoration: (node, output, state) => {
+    const {markdownMeta} = state
+    if (!markdownMeta) {
+      throw new Error('markdownMeta unexpectedly empty')
+    }
+    // $FlowIssue Don't know what's going on here.
+    const {message} = markdownMeta
```

Comment:
anyone know what's going on here? `message` isn't a key in `markdownMeta` (which itself may be null)

Reply:
It is, my guess is this PR breaks in-chat wallet sends.

Reply:
The code doesn't actually change any logic (it just silences the flow errors), but I'll look at the in-chat wallet send flow. Thanks!